packages/components/package.json

```diff
@@ -77,7 +77,7 @@
     "@types/react-datepicker": "2.9.3",
     "@types/react-router": "3.0.13",
     "@types/react-select": "1.0.57",
-    "@types/storybook__react": "4.0.1",
+    "@types/react-test-renderer": "^16.9.1",
```

Comment:
Do we want to pin a specific version here for @types/react-test-renderer or keep the `^`?

Reply:
Intellij did that, I'll pin it.